releases/release-1.30/release-notes/maps/pr-112957-map.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 112957
+releasenote:
+  text: |
+    "kubelet allowed specifying a custom root directory for pod logs (instead of the default /var/log/pods) using the `podLogsDir` key in kubelet configuration."
+pr_body: ""
```

Comment:
Hm, did you use the latest `krel` version here? The PR body should not be empty :thinking:

Reply:
Yes, installed the latest version `v0.16.7` using this command `go install k8s.io/release/cmd/krel@latest`

Reply:
@fykaa that's odd:

```
> krel release-notes --fix --fork saschagrunert --maps-from ~/go/src/k8s.io/sig-release/releases/release-1.30/release-notes --tag v1.30.0-rc.1 --create-draft-pr
INFO Checking if a PR can be created from saschagrunert/sig-release
Create draft pull request? (Y/n) (1/10)
Y
INFO Gathering release notes from v1.29.0 to v1.30.0-rc.1
INFO Cloning/updating repository kubernetes/kubernetes
INFO Using found start SHA: 3f7a50f38688eb332e2a1b013678c6435d539ae6
INFO Using found end SHA: 77aa9c21c295614817d5d49ea81f20783480691b
INFO Using output format: markdown
INFO Using start tag v1.29.0
INFO Using end tag v1.30.0-rc.1
INFO Gathering release notes
INFO Starting to process commit 1 of 1834 (0.05%): ec301a5cc76f48cdadc77bcfbd686cf40b124ecf
…
INFO PR #122435 seems to contain a release note
INFO Checking PRs for mapped data
INFO Successfully parsed release notes maps for 174 PRs from /home/sascha/go/src/k8s.io/sig-release/releases/release-1.30/release-notes
INFO Finished gathering release notes in 2m12.39699112s
INFO Cloning/updating repository kubernetes/sig-release
Press enter to start (1/10)

INFO Read 982 PR reviews from previous sessions

Welcome to the Kubernetes Release Notes editing tool!

This tool will allow you to review and edit all the release
notes submitted by the Kubernetes contributors before publishing
the updated draft.

The flow will show each of the release notes that need to be
reviewed once and you can choose to edit it or not.

After you choose, it will be marked as reviewed and will not
be shown during the next sessions unless you choose to do a
full review of all notes.

You can hit Ctrl+C at any time to exit the review process
and submit the draft PR with the revisions made so far.

Would you like to continue from the last session? (Y/n) (1/10)
Y

Release Note for PR 123384:
===========================
Pull Request URL: https://github.com/kubernetes/kubernetes/pull/123384
INFO Successfully parsed release notes maps for 174 PRs from /tmp/k8s-485508532/releases/release-1.30/release-notes
✨ Note contents was previously modified with a map
    Author: @aleksandra-malinowska
    SIGs: [scheduling]
    Kinds: [bug]
    Areas: []
    Feature: false
    ActionRequired: false
    DoNotPublish: false
    Text:
    │ Improved scheduler performance when no scoring plugins are defined.

- Fix note for PR #123384? (y/N) (1/10)
y
INFO Opening file with editor [vim /tmp/release-notes-map-3169703535.yaml]
```

Then editing the release note without setting any `pr_body` field.

Cancelling the edit session:
```
^CINFO Input cancelled, exiting edit flow
INFO Successfully parsed release notes maps for 174 PRs from /tmp/k8s-485508532/releases/release-1.30/release-notes/maps
INFO Release Notes Markdown Draft written to /tmp/k8s-485508532/releases/release-1.30/release-notes/release-notes-draft.md
INFO Release Notes JSON version written to /tmp/k8s-485508532/releases/release-1.30/release-notes/release-notes-draft.json
Create pull request with your changes? (y/n) (1/10)
^CFATA creating Draft PR: while asking to create pull request: Input canceled
```

Then the release note should contain the right `pr_body` value:
```
> cat /tmp/k8s-485508532/releases/release-1.30/release-notes/maps/pr-123384-map.yaml
───────┬────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
       │ File: releases/release-1.30/release-notes/maps/pr-123384-map.yaml
       │ Size: 1.2 KB
───────┼────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
   1   │ pr: 123384
   2   │ releasenote:
   3 ~ │   text: Foo bar
   4 ~ │ pr_body: "Resubmit of #122435, reverted  in #123290 due to increasing the chance of
   5 ~ │   not finding an existing feasible node when extender filters are used. The issue
   6 ~ │   itself is pre-existing and it's not clear whether it is something to be fixed (or
   7 ~ │   how). This change is modified to exclude the cases when extenders are used.\r\n\r\n####
   8 ~ │   What type of PR is this?\r\n\r\n/kind bug\r\n\r\n#### What this PR does / why we
   9 ~ │   need it:\r\n\r\nSkip looking for extra nodes if no scoring plugins and no extenders
  10 ~ │   are defined. As there's no difference in priority between nodes when we don't score
  11 ~ │   them and there's no risk that some won't pass extender filters, there's no point
  12 ~ │   in searching for multiple fitting nodes.\r\n\r\n#### Which issue(s) this PR fixes:\r\n\r\nFixes
  13 ~ │   #122057\r\n\r\n#### Special notes for your reviewer:\r\n\r\n#### Does this PR introduce
  14 ~ │   a user-facing change?\r\n\r\n```release-note\r\nImproves scheduler performance when
  15 ~ │   no scoring plugins are defined.\r\n```\r\n\r\n#### Additional documentation e.g.,
  16 ~ │   KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:\r\n\r\n```docs\r\n\r\n```\r\n\r\n/sig
  17 ~ │   scheduling\r\n/cc @ahg-g"
───────┴────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────

```

Do you use linux?

Reply:
I use linux

Reply:
None of the [previous PRs](https://github.com/kubernetes/sig-release/pull/2459/files) seems to have a pr_body content. But neither do they have an extra line stating pr_body is empty

Reply:
Yes, that's a feature which has been recently added. It should be either not existing or non-empty. That should not break anything, but it's still strange.

Reply:
What do you think should we do about this now

Reply:
As @rashansmith commented. We'll create an issue about this problem in Linux. We'll move forward with this PR for now.